### ISSUE TITLE
From where are those files loaded?

### ISSUE THREAD

**Issue by trancephorm** [orbitdb/orbit-db#894]
Tried executing browser.html example locally, but I see some libraries are included, but don't see where they could be found?

    <script type="text/javascript" src="../../dist/orbitdb.js" charset="utf-8"></script>
    <script type="text/javascript" src="../../node_modules/ipfs/dist/index.min.js" charset="utf-8"></script>
    
I mean, what good is example when you can't even run it?

**Comment by phillmac** [comment]
The assumption is you have cloned the repo, run `npm install` and `npm run build`.
Then those files should exist in those specific paths.

**Comment by trancephorm** [comment]
Yeah I realised that in the meantime, but that's not then a browser example, it's nodejs? Sorry I'm new in this field, question may sound stupid. :)

**Comment by trancephorm** [comment]
I would suggest those two lines to be updated with
  <script type="text/javascript" src="https://unpkg.com/ipfs@0.55.2/dist/index.min.js"></script>
  <script type="text/javascript" src="https://unpkg.com/orbit-db@0.26.1/dist/orbitdb.min.js"></script>
